src/components/composer/composer/composer.ts

```diff
@@ -369,14 +369,22 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.processTokenAtCursor();
   }
 
-  onKeyup() {
+  onKeyup(ev: KeyboardEvent) {
     if (this.contentHelper.el === document.activeElement) {
+      const isSelectingForComposer = this.env.model.getters.isSelectingForComposer();
+      const isArrowKey = ["ArrowUp", "ArrowDown", "ArrowLeft", "ArrowRight"].includes(ev.key);
+      if (isSelectingForComposer && isArrowKey) {
```

Comment:
```suggestion
      if (isSelectingForComposer && ev.key.startsWith("Arrow")) {
```

Reply:
@rrahir, I have made the suggested changes :)